tpm_futurepcr/util.py

```diff
@@ -119,3 +120,22 @@ def find_mountpoint_by_partuuid(partuuid):
                          stdout=subprocess.PIPE)
     res.check_returncode()
     return res.stdout.splitlines()[0].decode()
+
+class KeyValueAction(argparse.Action):
+    def __init__(self, option_strings, dest, **kwargs):
+        return argparse.Action.__init__(self,
+                option_strings=option_strings,
+                dest=dest,
+                **kwargs)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        if not hasattr(namespace, self.dest) or getattr(namespace, self.dest) is None:
```

Comment:
I think this could be shortened to `if getattr(namespace, self.dest, None) is None`?

Reply:
Good point. I'll make that change, and squash the entire thing into one commit to keep the history clean.